test/AddressRegistry.Tests/BackOffice/Infrastructure/TestBackOfficeContext.cs

```diff
@@ -10,12 +10,17 @@ namespace AddressRegistry.Tests.BackOffice.Infrastructure
 
     public class TestBackOfficeContext : BackOfficeContext
     {
+        private readonly bool _dontDispose;
+
         // This needs to be here to please EF
         public TestBackOfficeContext() { }
 
         // This needs to be DbContextOptions<T> for Autofac!
-        public TestBackOfficeContext(DbContextOptions<BackOfficeContext> options)
-            : base(options) { }
+        public TestBackOfficeContext(DbContextOptions<BackOfficeContext> options, bool dontDispose = false)
```

Comment:
can we invert this to "dispose = true"?

dontdispose = false reads like a double negation